internal/services/databricks/validate/databricks_workspaces_id.go

```diff
@@ -0,0 +1,23 @@
+package validate
```

Comment:
this file shouldn't be needed anymore, since it's available within `go-azure-sdk`?

Reply:
Fixed.